toar/toar_planetscope.ipynb

```diff
@@ -0,0 +1,474 @@
+    "\n",
+    "But over the course of a day or year, the number of photons that the sun shines on the scene rises and falls. If you naively compare the radiance values of two scenes over the same spot on Earth from the same satellite but a few weeks (or even just hours!) apart, you will likely find dramatic radiance differences **even if nothing on the ground has changed!**\n",
+    "\n",
+    "In addition to this variation, each of Planet Labs' 137 satllites have small amounts of variation in their spectral filters which yields slight differences in radiance measurements, even from two satellites taking pictures of the same exact place at the same exact moment!\n",
+    "\n",
+    "To correct all this radiance variation you would have to do a lot of math using the exact location and local time of day to find the angle to the sun and sun-earth distance, then compute a solar irradiance model to estimate how many photons of each band are present in the image, and finally convolve that spectrum with the spectral response of the individual satellite to yield the number of photons of each band that are actually recorded by that sensor. Dividing by this number normalizes the measured brightness to the brightness of the sun at that time and place through that particular filter, yielding a much more comparable number: reflectance.\n",
+    "\n",
+    "Top of Atmosphere Reflectance is extremely useful because it is an apples-to-apples comparable number from any satellite over any location that does not change with time of day or time of year **unless the content of the scene changes.** It is very commonly used in GIS applications which compute spectral indices such as NDVI or NDWI. \n",
+    "\n",
+    "It is so broadly useful that Planet does this math for you and provides all the coefficients necessary to convert a radiance image into a reflectance image!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\n",
+    "\\begin{equation*}\n",
+    "\\mathbf{img}_{reflectance} =  \\begin{vmatrix}\n",
+    "a \\\\\n",
+    "b \\\\\n",
+    "c \\\\\n",
+    "d \n",
+    "\\end{vmatrix}\n",
+    "\\times \\mathbf{img}_{radiance}\n",
+    "\\end{equation*}\n",
+    "$$\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The four coefficients $a, b, c, d$ are calculated and provided with every analytic image that Planet provides and can be used as simple scaling factors to convert from radiance to reflectance. Their values change with the image's local \n",
+    "time of day and time of year, and do so uniquely per satellite."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**In this guide, you'll perform a basic Radiance to Reflectance calculation on PlanetScope imagery using just a few lines of Python. Here are the steps:**\n",
+    "\n",
+    "1. Download a PlanetScope image\n",
+    "2. Extract data from each color band\n",
```

Comment:
Might want to use 'spectral band' rather than color band here, with analytic products we tend to avoid the use of the word color